app/api/src/services/projects/projects.ts

```diff
@@ -244,7 +245,19 @@ export const updateProjectImages = async ({
     const [updatedProject] = await Promise.all([
       projectPromise,
       imageDestroyPromise,
-    ]).then(() => sendChat(`project image updated: ${id}`));
+    ]).then(async (result) => {
+      const { userName } = await db.user.findUnique({
+        where: { id: project.userId },
+      })
+      sendDiscordMessage([
+        `${userName} just added an image to their ${project.cadPackage} project:`,
+        `  =>  ${project.title}`,
+        ``,
+        `Check it out, leave a comment, make them feel welcome!`,
+        `https://cadhub.xyz/u/${userName}/${project.title}`
+      ].join('\n'), `https://res.cloudinary.com/irevdev/image/upload/c_scale,w_700/v1/${mainImage}`)
+      return result
```

Comment:
The result needs to be return since `const [updatedProject] = await Promise.all([` is expecting the result.